sample.env

```diff
@@ -0,0 +1,11 @@
+# See https://docs.octopize.io/docs/deploying/configuration
```

Comment:
In `Readme`, you say to refer to the link provided by Octopize. Here, you give it. What do we really want to do ?